image_tf_encoder.py

```diff
@@ -51,7 +51,7 @@ def __init__(self,
                  pool_strategy: str = 'max',
                  default_batch_size: int = 32,
                  default_traversal_paths: List[str] = ['r'],
-                 on_gpu: bool = False,
+                 on_gpu: bool = True,
```

Comment:
why the default is `True`?  GPU is not always available right?

Reply:
The thing is that otherwise the GPU dockerfile does not work correctly. The entrypoint uses config.yml as configuration and this config.yml does not set any parameters. Therefore we need to set this to true to make sure the GPU docker container works correctly.
It also works for the other cases, there is only a warning because no GPU was found.

There might be better fixes for the problem, however.

Reply:
emmm, ok, no problem for me, just curious